src/arch-riscv64.c

```diff
@@ -0,0 +1,17 @@
+#include <stdlib.h>
```

Comment:
Do we want some sort of copyright header here?  If nothing else perhaps the standard LGPL boilerplate?